api/api.proto

```diff
@@ -4106,6 +4109,30 @@ message CollectDiagsJob {
   repeated DiagsCollectionStatus statuses = 2;
 }
 
+// DefragJob describes a job to run defragmentation on cluster nodes
+message DefragJob {
+  // MaxDurationHours defines the time limit in hours 
+  double max_duration_hours = 1;
+  // MaxNodesInParallel defines the maximum number of nodes running the defrag job in parallel
+  uint32 max_nodes_in_parallel = 2;
+  // IncludeNodes is a list of node UUID: if provided, will only run the job on these nodes;
+  // if not provided, will run on all nodes
+  // cannot coexist with ExcludeNodes and NodeSelector
+  repeated string include_nodes = 3;
+  // ExcludeNodes is a list of node UUID: if provided, the job will skip these nodes;
+  // if not provided, will run on all nodes
+  // cannot coexist with IncludeNodes
+  repeated string exclude_nodes = 4;
+  // NodeSelector is a list of node label `key=value` pairs separated by comma, 
+  // which selects the nodes to be run on for the job
+  // can coexist with ExcludeNodes but cannot coexist with IncludeNodes
+  repeated string node_selector = 5;
+  // RunningNodes stores the nodes on which the job is currently running
+  repeated string running_nodes = 6;
```

Comment:
for more context I would name this `current_running_nodes`.

Reply:
done